=== modified file 'src/provisioningserver/power_schema.py'
--- src/provisioningserver/power_schema.py	2014-04-23 14:21:03 +0000
+++ src/provisioningserver/power_schema.py	2014-04-29 20:40:05 +0000
@@ -242,4 +242,11 @@
             make_json_field('power_pass', "Power password"),
         ],
     },
+    {
+        'name': 'vmrunapi',
+        'description': 'VMWare Workstation / Fusion',
+        'fields': [
+            make_json_field('mac_address', "VM mac address"),
+        ],
+    },
 ]

